setup.py

```diff
@@ -15,7 +15,7 @@
 from setuptools import setup, find_packages
 
 NAME = "purity-fb"
-VERSION = "1.8.0"
+VERSION = "1.9"
```

Comment:
this is the package version so can we make it "1.9.0" in case we need to fix bugs in the client?